src/lib/Scenes/Fair/__tests__/Fair-tests.tsx

```diff
@@ -25,7 +25,7 @@ describe("Fair", () => {
           }
         `}
         mockData={{
-          Fair: () => fairFixture,
+          fair: fairFixture,
```

Comment:
Here is an example of a test that is now working properly with the updated `mockRelayRenderer`